components/TrackingComplaint/Detail/index.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div class="w-full">
+    <TrackingComplaintNoData v-show="false" />
+
+    <!-- Detail Complaint -->
+    <section class="flex flex-col gap-3">
+      <!-- Image -->
+      <div class="flex flex-col gap-3">
+        <div class="max-w-[900px] max-h-fit self-center">
+          <!-- Image Preview -->
+          <img
+            loading="eager"
+            :src="selectedImage"
+            width="900"
+            height="290"
+            class="rounded-lg mx-auto object-cover"
+            alt="gambar aduan"
+          />
+        </div>
+        <div class="w-full overflow-x-auto flex flex-row gap-1">
+          <button
+            v-for="(image, index) in images"
+            :key="'gambar' + index"
+            @click="selectedImage = image"
+          >
+            <img
+              loading="eager"
+              :src="image"
+              width="105"
+              height="65"
+              class="rounded-md max-w-[105px] max-h-[65px] object-cover"
+              :class="{
+                'opacity-50 transition-opacity duration-300':
+                  selectedImage === image,
+              }"
+              alt="gambar aduan"
+            />
+          </button>
+        </div>
+      </div>
+
+      <hr class="my-3" />
+
+      <div class="flex flex-col gap-1">
+        <h3 class="font-lato text-[12px] leading-[18px] text-gray-500">
+          Kategori Aduan
+        </h3>
```

Comment:
Why not use `text-xs` ? I see that many other places you also use arbitrary values ​​like this

Reply:
[fix here](https://github.com/jabardigitalservice/super-app-web/pull/96/commits/0ddf3f657b6cf2393215b3a90572424fc1de1ac3)